AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginIntegrationTests/DataStoreObserveQueryTests.swift

```diff
@@ -344,17 +409,17 @@ class DataStoreObserveQueryTests: SyncEngineIntegrationTestBase {
     ///
     func testSortWithCreateUpdateDelete() async throws {
```

Comment:
`testSortWithCreateUpdateDelete `: Now that we have the AsyncExpectation types, let's fix this V2 test back to what V1 looks like 
<img width="566" alt="image" src="https://user-images.githubusercontent.com/1365977/193640928-2786e4f6-3d2f-4c30-a054-04dfa598e4ae.png">